src/server/templates/typescript.ts

```diff
@@ -9,6 +9,8 @@ import type {
 } from '../../lib/index.js'
 import type { GeneratorMetadata } from '../../lib/generators.js'
 
+const defaultSchema = process.env.PG_META_GENERATE_TYPES_DEFAULT_SCHEMA
```

Comment:
This should be in `src/server/constants.ts`

Reply:
Fixed